aiounifi/interfaces/vouchers.py

```diff
@@ -0,0 +1,43 @@
+"""Hotspot vouchers as part of a UniFi network."""
+
+from ..models.api import TypedApiResponse
+from ..models.voucher import (
+    Voucher,
+    VoucherCreateRequest,
+    VoucherDeleteRequest,
+    VoucherListRequest,
+)
+from .api_handlers import APIHandler
+
+
+class Vouchers(APIHandler[Voucher]):
+    """Represents Hotspot vouchers."""
+
+    obj_id_key = "_id"
+    item_cls = Voucher
+    api_request = VoucherListRequest.create()
+
+    async def create(self, voucher: Voucher) -> TypedApiResponse:
+        """Create voucher on controller."""
+        return await self.controller.request(
+            VoucherCreateRequest.create(
+                number=1,
+                quota=voucher.quota,
+                expire_number=int(
+                    voucher.duration.total_seconds() / 60  # Get minutes.
+                ),
+                expire_unit=1,
```

Comment:
As the default value is 1 for expire unit would this be ok, or is there a specific reason this is 1
```suggestion
                expire_unit=voucher.expire_unit,
```

Reply:
exire_unit is not an attribute of Voucher. 1 = minute, because expire_number came from voucher.duration in Seconds / 60 = Minutes

The attributes for vouchers of the return values of the API are called differently than those for creating.
For example, when creating, you pass the number of vouchers to be created. But these are then created as separate vouchers.
And when creating, you specify the expiry time and in which unit you specify the expiry time. In the return, you always get the duration time in seconds. However, you cannot create a voucher in seconds ....

Reply:
Alright, yes indeed, thanks, as it has the default value assigned this can just be removed.
```suggestion
```